tests/interface/TWHashTests.cpp

```diff
@@ -237,3 +237,81 @@ TEST(TWHashTests, XXHash64Concat) {
+    }
+}
+
+TEST(TWHashTests, SHA256RIPEMD) {
+    auto tests = {
+        make_tuple(string(""), string("b472a266d0bd89c13706a4132ccfb16f7c3b9fcb")),
+        make_tuple(brownFox, string("0e3397b4abc7a382b3ea2365883c3c7ca5f07600")),
+        make_tuple(brownFoxDot, string("49f3b08d52a030415a5f7e84f72386e10acc4ebf")),
+    };
+    for (auto &test: tests) {
+        const auto inData = WRAPD(TWDataCreateWithBytes(reinterpret_cast<const uint8_t *>(get<0>(test).c_str()), get<0>(test).length()));
+        const auto hash = WRAPD(TWHashSHA256RIPEMD(inData.get()));
+        EXPECT_EQ(hex(data(TWDataBytes(hash.get()), TWDataSize(hash.get()))), get<1>(test));
+    }
+}
+
+TEST(TWHashTests, SHA3_256RIPEMD) {
+    auto tests = {
+        make_tuple(string(""), string("b0a2c9108b9cff7f0f686fef1d2ecbd5f1999972")),
+        make_tuple(brownFox, string("e70a0c74dd1b0c0d5af3c7ccbbe4b488d1b474b5")),
+        make_tuple(brownFoxDot, string("e88da28af1e8ded701c54eba19d10b79e7828d1c")),
+    };
+    for (auto &test: tests) {
+        const auto inData = WRAPD(TWDataCreateWithBytes(reinterpret_cast<const uint8_t *>(get<0>(test).c_str()), get<0>(test).length()));
+        const auto hash = WRAPD(TWHashSHA3_256RIPEMD(inData.get()));
+        EXPECT_EQ(hex(data(TWDataBytes(hash.get()), TWDataSize(hash.get()))), get<1>(test));
+    }
+}
+
+TEST(TWHashTests, Blake256Blake256) {
+    auto tests = {
+        make_tuple(string(""), string("d8ee5f957b78a961fb729098b4efb56440a14e05e3c55890f5edbc626380aaa6")),
+        make_tuple(brownFox, string("4511ab8713d8d580cae73061345df903f603b99e7ec699ddae63c56eea200059")),
+        make_tuple(brownFoxDot, string("f776b4c84c298bb8b1fdd0303eca63071611b9fa354991626f1a06430a6890bf")),
+    };
+    for (auto &test: tests) {
+        const auto inData = WRAPD(TWDataCreateWithBytes(reinterpret_cast<const uint8_t *>(get<0>(test).c_str()), get<0>(test).length()));
+        const auto hash = WRAPD(TWHashBlake256Blake256(inData.get()));
+        EXPECT_EQ(hex(data(TWDataBytes(hash.get()), TWDataSize(hash.get()))), get<1>(test));
+    }
+}
+
+TEST(TWHashTests, Blake256RIPEMD) {
+    auto tests = {
+        make_tuple(string(""), string("413320bee32a3bdb92b145d337316739d54a9287")),
+        make_tuple(brownFox, string("b4b44de1e854f7f3c0520b654204163f75f704e5")),
+        make_tuple(brownFoxDot, string("a2561519857a11175d1a85c424104aaeff6ccffd")),
+    };
+    for (auto &test: tests) {
+        const auto inData = WRAPD(TWDataCreateWithBytes(reinterpret_cast<const uint8_t *>(get<0>(test).c_str()), get<0>(test).length()));
+        const auto hash = WRAPD(TWHashBlake256RIPEMD(inData.get()));
+        EXPECT_EQ(hex(data(TWDataBytes(hash.get()), TWDataSize(hash.get()))), get<1>(test));
+    }
+}
+
+TEST(TWHashTests, Groestl512Groestl512) {
+    auto tests = {
+        make_tuple(string(""), string("fdfb14d386c6dff85715c50efb826c43e04205b18410497aa47f121eceb3a65e36a4671c728176311b4341d1d8d1d335822b9ed4b12a26e9c907557b4fcb385e")),
```

Comment:
Did you consider adding some tests on Swift/Kotlin? not all, just a few would be great